hw03_stubs.py

```diff
@@ -16,6 +16,9 @@
 #
 #################################################################################
 
+
+
+
```

Comment:
You were not supposed to edit this file! We will leave it, but make sure you aren't editing outside of your expected files; they might cause merge conflicts for others.